source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomActionOpenURLRenderer.mm

```diff
@@ -28,12 +28,31 @@ - (UIButton* )renderButton:(UIViewController *)vc
                                              superview:superview
                                      baseActionElement:acoElem
                                             hostConfig:acoConfig];
+    NSString *userSpecifiedBackgroundColor = nil;
     if(button)
     {
-        [button setBackgroundColor:[UIColor colorWithRed:1.0
-                                                   green:59.0/255.0
-                                                    blue:48.0/255.0
-                                                   alpha:1.0]];
+        NSData *additionalProperty = [acoElem additionalProperty];
+        if(additionalProperty) {
+            NSDictionary *dictionary = [NSJSONSerialization JSONObjectWithData:additionalProperty options:NSJSONReadingMutableLeaves error:nil];
+            userSpecifiedBackgroundColor = dictionary[@"color"];
+            NSLog(@"%@", userSpecifiedBackgroundColor);
```

Comment:
>NSLog(@"%@", userSpecifiedBackgroundColor); [](start = 12, length = 43)

Did you mean to leave this line here?

Reply:
sorry, thanks for catching this!

---
In reply to: [171957088](https://github.com/Microsoft/AdaptiveCards/pull/1179#discussion_r171957088) [](ancestors = 171957088)